acurl/acurl/__init__.py

```diff
@@ -252,6 +252,8 @@ def to_curl(self):
             + shlex.quote(self.url)
         )
 
+    as_curl = to_curl
```

Comment:
Why?

Reply:
This was only a quality of life alias for my own selfish reasons. The handful of times I've been in the --debugging console and tried to get a curl I've typed as_curl first. Happy to remove and spend the extra 2 seconds I spend every time I do this correcting myself (or learn faster).